components/filter/filter-overflow-group.js

```diff
@@ -105,9 +116,27 @@ class FilterOverflowGroup extends OverflowGroupMixin(RtlMixin(LitElement)) {
 				if (!filterSetValue) return;
 				filterSetValue.selected = change.selected;
 			});
+
+			this._getFilterParent(e.target.classList, filterSet)?.requestFilterChangeEvent(e.detail.allCleared, [dimension], { overflowEvent: true });
```

Comment:
When a filter change event happens on the overflow filter, the original filter that is currently hidden is then found and the change event is triggered on it.

Would need to test and make sure that if a change handler on the original filter changed something within that original filter that it is then reflected to overflow group filter.